addons/stock/static/src/js/counted_quantity_widget.js

```diff
@@ -0,0 +1,37 @@
+odoo.define('stock.counted_quantity_widget', function (require) {
+    'use strict';
+
+const BasicFields = require('web.basic_fields');
+const fieldRegistry = require('web.field_registry');
+
+const CountedQuantityWidgetField = BasicFields.FieldFloat.extend({
+    supportedFieldTypes: ['float'],
+
+     _renderReadonly: function () {
+        if (this.recordData.inventory_entered) {
+            this.$el.text(this._formatValue(this.recordData.inventory_quantity));
+        } else {
+            this.$el.text("");
+        }
+    },
+
+    _onChange: function () {
+        this._rpc({
```

Comment:
Is it possible to check if the record data if the quantity is already set to avoid too much rpc